components/script/dom/webidls/Notification.webidl

```diff
@@ -1,15 +1,8 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-/*
- * The origin of this IDL file is
- * https://notifications.spec.whatwg.org/
- *
- * Copyright:
- * To the extent possible under law, the editors have waived all copyright and
- * related or neighboring rights to this work.
- */
 
+// https://notifications.spec.whatwg.org/#api
```

Comment:
You probably need to add this URL to https://github.com/servo/servo/blob/main/python/tidy/tidy.py#L76

Reply:
It is! Thanks!